dev/archery/archery/docker/cli.py

```diff
@@ -216,7 +216,8 @@ def docker_run(obj, image, command, *, env, user, force_pull, force_build,
     PYTHON=3.8 archery docker run conda-python
 
     # disable the cache only for the leaf image
-    PANDAS=master archery docker run --no-leaf-cache conda-python-pandas
+    PANDAS=upstream_devel archery docker run --no-leaf-cache
+    conda-python-pandas
```

Comment:
Either put this on a single line or add escape the newline:
```suggestion
    PANDAS=upstream_devel archery docker run --no-leaf-cache conda-python-pandas
```

```suggestion
    PANDAS=upstream_devel archery docker run --no-leaf-cache \
        conda-python-pandas
```

(with a shorter name than "upstream_devel" we probably wouldn't have this problem ;-))